README.md

```diff
@@ -1,20 +1,20 @@
 <div align="center">
-  <a aria-label="Story of AMS logo" href="https://storyofams.com/" target="_blank" align="center">
-    <img src="https://avatars.githubusercontent.com/u/19343504" alt="Story of AMS" width="100">
+  <a aria-label="Instant Commerce logo" href="https://instantcommerce.io/" target="_blank" align="center">
+    <img src="https://avatars.githubusercontent.com/u/93975473" alt="Instant Commerce" width="100">
   </a>
-  <h1 align="center">@storyofams/next-api-decorators</h1>
+  <h1 align="center">next-api-decorators</h1>
   <p align="center">
-    <a aria-label="releases" href="https://GitHub.com/storyofams/next-api-decorators/releases/" target="_blank">
-      <img src="https://github.com/storyofams/next-api-decorators/workflows/Release/badge.svg">
+    <a aria-label="releases" href="https://GitHub.com/instantcommerce/next-api-decorators/releases/" target="_blank">
```

Comment:
Is there a reason for `GitHub` to be capitalized here?

Reply:
i don't there's any difference. it's been like that since the beginning.